docs/features/features-walk-through/metadata_only.md

```diff
@@ -0,0 +1,10 @@
+# Metadata-only records
+
+
+A metadata-only record is a record that contains only descriptive metadata, without any associated files or content. These types of records can be useful in cases where you want to provide metadata for a resource that does not have a corresponding digital object or content file.
```

Comment:
```suggestion
A metadata-only record is a record that contains only descriptive metadata, without any associated files. These types of records can be useful in cases where you want to provide metadata for a resource that does not have a corresponding digital object or content file.
```

The two sentences seem a bit redundant to me, but I dont have any suggestions :/